src/applications/pensions/components/HomeAcreageValueReview.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isValidCurrency } from '../validation';
+import { isHomeAcreageMoreThanTwo, formatCurrency } from '../helpers';
+
+export const content = {
+  label: 'What’s the value of the land that’s more than 2 acres?',
+  error: 'Please enter a valid dollar amount',
+  title: 'Home acreage value',
+  edit: 'Edit',
+};
+
+const HomeAcreageValueReview = ({ data, editPage }) => {
+  const { homeAcreageValue = '' } = data;
+  const error =
+    homeAcreageValue !== '' && !isValidCurrency(homeAcreageValue) ? (
+      <strong className="usa-input-error-message">{content.error}</strong>
+    ) : null;
+  const labelWrapClasses = error
+    ? 'vads-u-border-left--4px vads-u-border-color--secondary-dark vads-u-padding-left--1p5'
+    : '';
+
+  return isHomeAcreageMoreThanTwo(data) ? (
+    <div className="form-review-panel-page pensions-custom-review">
```

Comment:
I'd rather use a utility class here instead of adding a custom class. This should be close enough and better for maintainability.
```suggestion
    <div className="form-review-panel-page vads-u-margin-bottom--7">
```